charts/common/values.yaml

```diff
@@ -210,13 +210,20 @@ components:
     networkpolicy:
       # deploy has to be set to true for rendering to be applied
       deploy: false
+      # Optional IP CIDR source ranges to allow access to ingress
```

Comment:
write `to allow ingress traffic`, to prevent misunderstandings with the ingress concept